src/main/java/org/jabref/model/entry/LinkedFile.java

```diff
@@ -17,14 +17,16 @@
 import javafx.beans.property.SimpleStringProperty;
 import javafx.beans.property.StringProperty;
 
+import org.jabref.architecture.AllowedToUseLogic;
+import org.jabref.logic.util.FileUtil;
 import org.jabref.model.database.BibDatabaseContext;
-import org.jabref.model.util.FileHelper;
 import org.jabref.preferences.FilePreferences;
 
 /**
  * Represents the link to an external file (e.g. associated PDF file).
  * This class is {@link Serializable} which is needed for drag and drop in gui
  */
+@AllowedToUseLogic("Uses FileUtil from logic")
```

Comment:
Can FileUtil be moved to the model domain?

Reply:
nope, tried this before not possible

Reply:
And handling files is really a "logic" thing, not a model thing. We can talk about refactoring the "model" package as some things are do not sound like "model"

![image](https://user-images.githubusercontent.com/1366654/229290193-e00aef2c-ee0e-4097-ba5d-0d1cc7fba7d9.png)